assets/src/blocks/CarouselHeader/CarouselHeaderScript.js

```diff
@@ -0,0 +1,11 @@
+// Hydratable components
+import { CarouselHeaderFrontend } from './CarouselHeaderFrontend';
+
+const BLOCK_NAME = 'planet4-blocks/carousel-header';
+
+document.querySelectorAll( `[data-hydrate="${BLOCK_NAME}"]` ).forEach(
```

Comment:
This won't be looping as the bundled one does (the `data-render` loop). But we could probably extract this into a function and import it on each frontend block script.

Reply:
Indeed seems better to extract already :+1: